Cargo.toml

```diff
@@ -74,7 +74,7 @@ macos-system-configuration = ["dep:system-configuration"]
 
 # Experimental HTTP/3 client.
 # Disabled while waiting for quinn to upgrade.
-# http3 = ["rustls-tls-manual-roots", "dep:h3", "dep:h3-quinn", "dep:quinn", "dep:futures-channel"]
+ http3 = ["rustls-tls-manual-roots", "dep:h3", "dep:h3-quinn", "dep:quinn", "dep:futures-channel"]
```

Comment:
We were getting a ton of warnings because this feature doesn't exist, just re-enabling it to silence them